projects/core/src/lib/users/user-session/user-session.component.ts

```diff
@@ -0,0 +1,33 @@
+import {Component, OnDestroy} from '@angular/core';
+import {Subscription} from 'rxjs';
+import {Session} from '../session.model';
+import {UserService} from '../user.service';
+import {Clipboard} from '@angular/cdk/clipboard';
+
+@Component({
+    selector: 'geoengine-user-session',
+    templateUrl: './user-session.component.html',
+    styleUrls: ['./user-session.component.scss'],
+})
+export class UserSessionComponent implements OnDestroy {
+    protected session: Session | undefined;
+    private sessionStreamSubscription: Subscription;
+
+    constructor(protected readonly userService: UserService, protected clipboard: Clipboard) {
+        const subs = this.userService.getSessionStream().subscribe((session) => {
+            this.session = session;
+        });
+
+        this.sessionStreamSubscription = subs;
+    }
+
+    ngOnDestroy(): void {
+        this.sessionStreamSubscription.unsubscribe();
+    }
+
+    copySessionTokenToClipboard(): void {
+        if (this.session) {
+            this.clipboard.copy(this.session.sessionToken);
+        }
```

Comment:
Somehow, the copy button isn't working for me.

Reply:
[643615e](https://github.com/geo-engine/geoengine-ui/pull/585/commits/643615e44353906280602288cef60eceb5f0f443)